app/src/main/java/org/mozilla/fenix/datastore/Stores.kt

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.datastore
+
+import android.content.Context
+import androidx.datastore.core.DataStore
+import androidx.datastore.dataStore
+
+/**
+ * Application / process unique [DataStore] for IO operations related to Pocket recommended stories selected categories.
+ */
+internal val Context.pocketStoriesSelectedCategoriesStore: DataStore<SelectedPocketStoriesCategories> by dataStore(
```

Comment:
Nit: Reading the code I noticed that we are now overloading the term "Store", which can be a `lib-state` `Store` or a `DataStore`. Maybe we should make that more obvious and name those things and related parameters `*DataStore` instead of `*Store`? 🤔